ldm/simplet2i.py

```diff
@@ -279,7 +282,8 @@ def process_image(image,seed):
             self._set_sampler()
 
         tic = time.time()
-        torch.cuda.torch.cuda.reset_peak_memory_stats()
+        if torch.cuda.is_available():
+            torch.cuda.torch.cuda.reset_peak_memory_stats()
```

Comment:
Do we really need to have torch.cuda.torch.cuda here? It looks like a bug, and I think torch.cuda.reset_peak_memory_stats() works just as well. However, I'm holding up the merge for this.